index.html

```diff
@@ -4264,6 +4264,48 @@ <h3>Spectrum of Privacy</h3>
         </p>
       </section>
 
+      <section class="informative">
+        <h3>Software Trust Boundaries</h3>
+
+        <p>
+There are a variety of trust boundaries that exist in the
+<a href="#ecosystem-overview">ecosystem described by this specification</a>.
+Just like an individual using a web browser trusts the web browser not to betray
+their trust by uploading their personal information to a data broker, a similar
+trust exists between the roles in the ecosystem described by this specification
+and the software that operates on the behalf of each of those roles. For
+example,
+        </p>
+
+        <ul>
+          <li>
+An <a>issuer</a> trusts the issuer software it uses to not issue <a>verifiable
+credentials</a> without its permission.
+          </li>
+          <li>
+A <a>verifier</a> trusts the verifier software it uses to be truthful regarding
+the verification status of the <a>verifiable credentials</a> and
+<a>verifiable presentations</a> that it is checking.
+          </li>
+          <li>
+A <a>holder</a> trusts the digital wallet software it uses to not divulge
+information to a <a>verifier</a> without its consent.
+          </li>
+        </ul>
+
+        <p>
+The examples above are not exhaustive, and there can be a variety of other
+expectations that roles have of the software that they use to achieve their
+goals. The common thread between each of these expectations is that each
+role expects the software that it uses to not violate privacy expectations.
+That is, it is expected that the software will operate in the best interests
+of the role and a violation of those expectations is a violation of trust that
+will result in the software being replaced by something that does not violate
+that trust. Implementers are strongly advised to write software that does not
+violate the trust of the roles they are serving.
```

Comment:
worth having something on holders/verifiers being able to/auditing the trust boundaries they encounter?

Reply:
++ That or a trusted third party (e.g. governance framework auditor). These specs have caught flack for not acknowledging these things within the spec so this seems like a good reference point here to acknowledge it.

Reply:
Added in dbd6ac7089b6ec96e8e1cfc1725d08f5e906a02c.